src/main/java/runhistoryplus/patches/ScoreBreakdownRunHistoryPatch.java

```diff
@@ -0,0 +1,176 @@
+package runhistoryplus.patches;
+
+import basemod.ReflectionHacks;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.evacipated.cardcrawl.modthespire.patcher.PatchingException;
+import com.megacrit.cardcrawl.audio.MusicMaster;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.GameCursor;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.helpers.Hitbox;
+import com.megacrit.cardcrawl.helpers.TipHelper;
+import com.megacrit.cardcrawl.metrics.Metrics;
+import com.megacrit.cardcrawl.monsters.MonsterGroup;
+import com.megacrit.cardcrawl.screens.DeathScreen;
+import com.megacrit.cardcrawl.screens.GameOverScreen;
+import com.megacrit.cardcrawl.screens.GameOverStat;
+import com.megacrit.cardcrawl.screens.VictoryScreen;
+import com.megacrit.cardcrawl.screens.runHistory.ModIcons;
+import com.megacrit.cardcrawl.screens.runHistory.RunHistoryScreen;
+import com.megacrit.cardcrawl.screens.stats.RunData;
+import javassist.CannotCompileException;
+import javassist.NotFoundException;
+import javassist.CtBehavior;
+import javassist.CtClass;
+import javassist.CtField;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ScoreBreakdownRunHistoryPatch {
+    private static ArrayList<String> scoreBreakdown;
+    private static String scoreHeading;
+
+    private static void createScoreBreakdown(ArrayList<GameOverStat> stats) {
+        scoreBreakdown = new ArrayList<String>();
+        for (GameOverStat stat : stats) {
+            if (stat.label == null) {
+                break;
+            }
+            scoreBreakdown.add(stat.label + ": " + stat.value);
+        }
+    }
+
+
+    @SpirePatch(clz = CardCrawlGame.class, method = SpirePatch.CONSTRUCTOR)
+    public static class ScoreBreakdownField {
+        @SpireRawPatch
+        public static void addScoreBreakdownField(CtBehavior ctBehavior) throws NotFoundException, CannotCompileException {
+            CtClass runData = ctBehavior.getDeclaringClass().getClassPool().get(RunData.class.getName());
+            String fieldSource = "public java.util.List score_breakdown;";
```

Comment:
Just noting here that I looked over the format of this and understand that we're storing the string that gets generated by the final screen (DeathScreen/VictoryString). From having looked at the score calculation code before, that's the best we can do with how it's organized.

Doing it this way also means that we should catch any custom score categories that people have added, which is important -- as long as any custom scoring is done by patching createGameOverStats in DeathScreen/VictoryScreen, and it adds to the `stats` field, it'll work.